### PR TITLE
Clarifying for other language

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2299

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/how-to-format-your-csv-file-for-data-content-migration.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ https://sharepoint2013.com/sites/contosoteamsite/,DocumentLibraryName,DocLibrary
 >  *Do not* include a header row in your CSV file. The second example included headers to demonstrate the order of the fields. Remember to account for all six columns in the file, even if you are not needing a value for a given field. 
   
 > [!IMPORTANT]
-> If you use the standard out-of-the-box Document library ("Shared Documents"), you must use the internal name "Documents" as the placeholder value for the  *Source Document Library* (Column B) in your CSV file. If you enter "Shared Documents" in that column, you will receive an "invalid document library" error. 
+> If you use the standard out-of-the-box Document library ("Shared Documents"), you must use the internal name "Documents" as the placeholder value for the  *Source Document Library* (Column B) in your CSV file. If you enter "Shared Documents" in that column, you will receive an "invalid document library" error.
```
Comment (JohanFreelancer9) [inline migration/how-to-format-your-csv-file-for-data-content-migration.md:53]:
```suggestion
> If you use the standard out-of-the-box Document library ("Shared Documents"), you must use the internal name "Documents" as the placeholder value for the *Source Document Library* (Column B) in your CSV file. If you enter "Shared Documents" in that column, you will receive an "invalid document library" error.
```

File: migration/how-to-format-your-csv-file-for-data-content-migration.md [hunk 2]
```diff
@@ -50,7 +50,8 @@ https://sharepoint2013.com/sites/contosoteamsite/,DocumentLibraryName,DocLibrary
...
+> If the language of the destination SharePoint site is other than english, check the internal name of the "Shared Documents" Document library by browsing to https://contoso.sharepoint.com/sites/SampleSite/_layouts/15/viewlsts.aspx?view=14.
```
Comment (JohanFreelancer9) [inline migration/how-to-format-your-csv-file-for-data-content-migration.md:54]:
```suggestion
> If the language of the destination SharePoint site is other than English, check the internal name of the "Shared Documents" Document library by browsing to https://contoso.sharepoint.com/sites/SampleSite/_layouts/15/viewlsts.aspx?view=14.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**